### ISSUE TITLE
Actinomyces infection of the lacrimal system

### ISSUE THREAD

**Issue by MickeySegal** [obophenotype/human-phenotype-ontology#9941]
**Preferred term label:**
Actinomyces infection of the lacrimal system

**Synonyms**
Actinomyces israelii (anaerobic filamentous gram positive) culture of discharge or concretions

**Definition (free text, please give PubMed ID)**

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
Canaliculitis

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
@MickeySegal 
Would this be under "Bacterial culture of conjunctival sac secretions" or how is this finding observed?

**Comment by MickeySegal** [comment]
https://pubmed.ncbi.nlm.nih.gov/15221305/
"Actinomyces canaliculitis presents with epiphora, chronic purulent conjunctivitis, a palpably thickened canaliculus, and yellow punctual discharge. In suspect cases canuliculotomy and curettage should be performed, although canalicular reconstruction is generally unnecessary. Culture of discharge and concretions using PD Plus/F blood culture medium gave improved results over accepted norms. Fixation of smeared concretions on a slide in alcohol is simple and is diagnostic of the disease. "

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000792
- label: Positive lacrimal tract actinomyces culture
- parents: [Positive respiratory tract infectious agent test (HP:0430151)](https://hpo.jax.org/app/browse/term/HP:0430151)
- pmid: 
- def: Growth of Actinomyces israelii (anaerobic filamentous gram positive) culture of lacrimal discharge or concretions.
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/9941
- synonyms:
